.github/workflows/ecosystem.yml

```diff
@@ -0,0 +1,536 @@
+name: Eco
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  ecosystem:
+    strategy:
+      fail-fast: false
+      matrix:
+        cfg:
+
+          - runs-on: ubuntu-latest
+            python-version: "3.10"
+            mkl-version: "2021.4"
+            base-channel: "defaults"
+            cmargs: >
+              -D CMAKE_VERBOSE_MAKEFILE=OFF
+              -D BUILD_SHARED_LIBS=ON
+
+          - runs-on: macos-latest
+            python-version: "3.9"
+            mkl-version: "2021.4"
+            base-channel: "defaults"
+            target-sdk: "10.10"
+            cmargs: >
+              -D CMAKE_VERBOSE_MAKEFILE=OFF
+              -D BUILD_SHARED_LIBS=ON
+              -D OpenMP_C_FLAG="-fopenmp=libiomp5"
+              -D OpenMP_CXX_FLAG="-fopenmp=libiomp5"
+            # Notes:
+            # * libiomp5 is picking up naturally but left explicit as guide
+
+          - runs-on: windows-latest
+            python-version: "3.9"
+            mkl-version: "2021.4"
+            base-channel: "conda-forge"
+            cmargs: >
+              -D CMAKE_VERBOSE_MAKEFILE=OFF
+              -D BUILD_SHARED_LIBS=OFF
+              -D CMAKE_CXX_FLAGS="/arch:AVX /wd4018 /wd4101 /wd4996"
+              -D OpenMP_LIBRARY_DIRS="D:\a\psi4\psi4\iomp5md\conda\win\2019.1"
+            # Notes:
+            # * mkl-version: "2019.1" is longstanding. needs defaults for intel-openmp
```

Comment:
I'm confused here - `mkl-version` points to `2021.4`, `OpenMP_LIBRARY_DIRS` has `2019.1` in the path, and the comment in notes is also referring to `2019.1`. Could we at least make the comment more verbose?